### PR TITLE
rmLink stranded resources

### PR DESCRIPTION
**Pull request by ellennickles**
This PR proposes the following:
* centralize information for running models (local CPU, local GPU, and remote) that were stranded on unlinked pages created during a previous major update
* this centralization adds navigation in the sidebar that is consistent with the other Learn topic areas
* across multiple pages: 
   * introduces minor copy edits consistent with language used on other public pages
   * updates some language to make topics more accessible to non-technical audiences

Notes:
The old local CPU and local GPU pages are still live, but they are dead ends in the current Learn section configuration; nothing links to them. There are currently no Support Articles that link to these pages—so no broken links!

However, the following Medium Article links to the “stranded” local GPU page, which I propose we update to the general Learn URL once this PR is live: https://medium.com/runwayml/announcing-local-gpu-support-2ba757f9d2a8

### CODE REVIEW

File: docs/how-to/run-models.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Run Models in RunwayML
```
Comment (cvalenzuela) [inline docs/how-to/run-models.md]: Let's make this `Advanced` options

File: docs/how-to/use-models.md [hunk 2]
```diff
@@ -86,39 +86,11 @@ Each model has a set of options, located in the right sidebar, that you can modi
 </div>
 
 Depending on the model, there are several ways to run models in RunwayML:
+* **Run Models with Local CPU.** To run models locally, you need to download and install them individually. Some models require you to install Docker, a free and open-source software. If you need to install Docker, [follow these steps](http://localhost:4000/#/how-to/run-models?id=overview).
```
Comment (cvalenzuela) [inline docs/how-to/use-models.md]: Add `Advanced` options reference here

File: docs/_sidebar.md [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+* 🎨 Create
     * 🌱 **Model Possibilities**
         <!-- * [Overview]() -->
         * [Detect Objects](create-with-runwayml/detect.md)
         * [Transform Content](create-with-runwayml/transform.md)
         * [Generate Media](create-with-runwayml/generate.md)
         * [Chain Models](how-to/chain-models-together.md)
         * [Train Models - coming soon!]() 
-    * 🤝 **Networking**
+    * 🤝 **Connect to Models**
```
[the remark this answers is not in the document]
Comment (ellennickles) [reply docs/_sidebar.md]: I kept this as "Connect" for now

File: docs/how-to/run-models.md (hunk 1 above)
Comment (ellennickles) [reply docs/how-to/run-models.md]: updated this!

File: docs/how-to/use-models.md (hunk 2 above)
Comment (ellennickles) [reply docs/how-to/use-models.md]: Also made the updates to this page with the new commits!

**Comment by ellennickles** [comment]
@cvalenzuela merging this now with the new edits we discussed but we can make additional updates to fine-tune the language if necessary!

**Pull request merged**